src/app/create/codebases/codebases.component.html

```diff
@@ -1,3 +1,5 @@
 <div id="codebases" class="container content">
-  <h1>TODO</h1>
+  <!-- can someone please show me how to make this be a relative url for the current component's directory?
+       I really just want to use 'codebases.png', but I've been unable to figure out how to do that -->
+  <img src='/app/create/codebases/codebases.png' style="width:100%;">
```

Comment:
It's not going to work like that, as this is an SPA app, the path to the file is actually /.

The right answer is to put codebases.png in /src/app/assets/img/ and then use a path of /assets/img/codebases.png in the html.